plan/plan_builder_test.go

```diff
@@ -1004,6 +1005,159 @@ func TestSortRelationErrors(t *testing.T) {
 	assert.ErrorContains(t, err, "output mapping index out of range")
 }
 
+func TestProjectExpressions(t *testing.T) {
```

Comment:
New test, round-tripping some function expressions. This fails without this PR: the URI gets duplicated, and it uses simple function names and not compound ones